tests/test_helpers.py

```diff
@@ -17,9 +17,9 @@
       "--date", "2020-04-29", "-c", "afriend", "-d", "more work",
       "-m", "ascopatz", "-g", "SymPy-1.1"],
      "20ly_newprojectum has been added in projecta\n"),
-    (["helper", "a_proposal", "a new proposal", "2020-12-29",
-      "500.0","2020-09-22","3", "To destroy numbers",
-      "-a", "Kurt Godel", "-p", "Simon Billinge", "-f", "Columbia"],
+    (["helper", "a_proposal", "a new proposal","500.0","To destroy numbers",
```

Comment:
this helper is not very well tested.  At least I think we need two tests, a minimal set of inputs (to test the defaults are working correctly), and a maximal set (so we have coverage for everything).